packages/errors/src/index.ts

```diff
@@ -113,41 +113,36 @@ export const MissingBucketMap = createErrorType('MissingBucketMap');
  */
 export const ProviderNotFound = createErrorType('ProviderNotFound');
 
-export const FTPError = createErrorType('FTPError');
-
-export const PDRParsingError = createErrorType('PDRParsingError');
-
 export const ConnectionTimeout = createErrorType('ConnectionTimeout');
 
-export const HostNotFound = createErrorType('HostNotFound');
-
-export const FileNotFound = createErrorType('FileNotFound');
+export const CumulusMessageError = createErrorType('CumulusMessageError');
 
-export const InvalidChecksum = createErrorType('InvalidChecksum');
+export const DeletePublishedGranule = createErrorType('DeletePublishedGranule');
 
 export const DuplicateFile = createErrorType('DuplicateFile');
 
-export const UnexpectedFileSize = createErrorType('UnexpectedFileSize');
+export const EcsStartTaskError = createErrorType('EcsStartTaskError');
 
-/**
```

Comment:
These comments don't appear to match the error class.    Removing and re-alpha-ing the uncommented errors.

Reply:
I don't follow. Did you just alphabetize the contents of this file?

Reply:
Yes, and removed the block comment for this and for `DeletePublishedGranule` as they make no sense.